### ISSUE TITLE
Fixate the chapter validation

### ISSUE THREAD

**Issue by martinheidegger** [nodeschool/admin#6]
The validation of a chapter is important and should be done before we migrate all chapters. After our first migration continuous updates might become more problematic so it would be good to have a clear expectation what _a chapter **is**_. Right now the validation takes _only_ care of the chapter.json and events.json but I assume it could also take care of images/hexdec/logo/etc. 

This is a sort-of-meta-issue. Please reference sub-tasks here.

_Memo: It would be good to have a clear definition of when this issue is done._

**Comment by bnb** [comment]
Brainstorming here:

A chapter needs: 
- chapter.json - self-explanatory
- events.json - self-explanatory
- Code of Conduct - Good practice, and sets guidelines for what is and isn't acceptable.
- Chapter shortcode - a 3 or 4 (comments?) letter code that can be easily referred to instead of having to know the whole name, location, et al. More for people or computers?
- Hexagon logo - logo to be pulled automatically
- humans file - Avatars/names/positions/roles/etc. for organizers and maintainers? Humans.json is a project I've wanted to work on and have started a few times, including another attempt just the other day. Would love to help implement a validator for it/spin it off here. Also have humansjson.com.

Just ideas. I can also set up a file structure, if requested.

**Comment by martinheidegger** [comment]
**Code of Conduct**: I think its better to have a global code-of-conduct at nodeschool to reference to. Would love to have input from @maxogden @mikeal on this

**Chapter shortcode**: See #13 

**Hexagon Logo**: See #12 (should be optional - right?)

**humans file**: I think the github teams are a better place to keep this else we will again run out-of-sync like the current organisers array in the chapter.json (Public Github account information is quite detailed)

**Comment by max-mapper** [comment]
one thing to note about github org teams is that it requires org membership to view, e.g. open https://github.com/orgs/nodeschool/teams/augsburg when logged in as a user not in the /nodeschool org and you get a 404

**Comment by SomeoneWeird** [comment]
Maybe another admin tool we should make in that case is something to sync team members with a json(?) file in a public repo.

**Comment by martinheidegger** [comment]
@maxogden Right! I completely forgot about that. In this case I think it would be "easiest" to setup a webserver that does nothing but show the content of a team.

Linking: #18
